make_emscripten.sh

```diff
@@ -4,35 +4,45 @@ set -e
 rm -rf emscripten/*
 mkdir -p emscripten
 
-cmake .
```

Comment:
I think this is required, why remove it?

Reply:
It was not necessary (for me).

Reply:
I think we need to run cmake to generate some files - you might have run it before

Reply:
Yeah, thats possible. Let me update this PR.